### PR TITLE
Update boroughs of Paris

### PR DESCRIPTION
**Pull request by stepps00**
This PR adds French and English preferred and variant names to the borough records in Paris (arrondissements), as well as Wikidata concordances. No PIP work needed, just a property addition.

### CODE REVIEW

File: data/115/889/427/3/1158894273.geojson [hunk 1]
```diff
@@ -20,23 +20,36 @@
...
+    "name:eng_x_variant":[
+        "20th arrondissement of Paris"
+    ],
+    "name:fra_x_preferred":[
         "20e Arrondissement"
     ],
+    "name:fra_x_variant":[
+        "20e arrondissement de paris",
+        "XXe arrondissement de paris",
+        "Paris XXe"
```
Comment (nvkelso) [inline data/115/889/427/3/1158894273.geojson:35]: Would it be crazy to set the `"label:fra_x_preferred"` to `"Paris XXe"` to shorten what is displayed for reverse geocoding?

**Review by nvkelso — APPROVED** [review]
LGTM

One question about if we should also be setting `label` properties for these.

File: data/115/889/427/3/1158894273.geojson (hunk 1 above)
Comment (stepps00) [reply data/115/889/427/3/1158894273.geojson:35]: Hmm.. let me poke around, I'll come back to this one...

**Comment by stepps00** [comment]
I'm going to merge this as-is, but will file a follow up PR after researching.

**Pull request merged**